docs/verticalpodautoscaler-metrics.md

```diff
@@ -30,9 +30,9 @@ spec:
             gauge:
               path: [metadata, annotations]
 # This will output the following metric:
-# HELP kube_crd_autoscaling_k8s_io_v1_VerticalPodAutoscaler_annotations Kubernetes annotations converted to Prometheus labels.
-# TYPE kube_crd_autoscaling_k8s_io_v1_VerticalPodAutoscaler_annotations gauge
-# kube_crd_autoscaling_k8s_io_v1_VerticalPodAutoscaler_annotations{namespace="default",target_api_version="autoscaling.k8s.io/v1",target_kind="Deployment",target_name="hamster",verticalpodautoscaler="hamster-vpa"} 123
+# HELP kube_customresource_autoscaling_k8s_io_v1_VerticalPodAutoscaler_annotations Kubernetes annotations converted to Prometheus labels.
```

Comment:
It looks like these also use the old naming scheme which we replaced (and this MR updates its fields).

I believe they should look like this: 
```
# HELP kube_customresource_annotations Kubernetes annotations converted to Prometheus labels.
# TYPE kube_customresource_autoscaling_k8s_io_v1_VerticalPodAutoscaler_annotations gauge
# kube_customresource_annotations{customresource_group="autoscaling.k8s.io", customresource_version="v1", customresource_kind="VerticalPodAutoscaler", namespace="default",target_api_version="autoscaling.k8s.io/v1",target_kind="Deployment",target_name="hamster",verticalpodautoscaler="hamster-vpa"} 123
```